lib/components/flat_button_short.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/design/app_colors.dart';
+import 'package:seeds/domain-shared/ui_constants.dart';
+
+/// A short flat widget button with rounded corners
+class FlatButtonShort extends StatelessWidget {
+  final String title;
+  final VoidCallback? onPressed;
+  final bool enabled;
+  final bool isLoading;
```

Comment:
this is not used.

Reply:
is not being used but it should be use, is a button so it should have the same functionalities as the other ones